.travis.yml

```diff
@@ -1,38 +1,59 @@
 language: php
+sudo: false
+cache:
+    directories:
+        - $HOME/.composer/cache/files
+        - $HOME/symfony-bridge/.phpunit
 
-php:
-    - 5.4
-    - 5.5
-    - 5.6
-    - 7.0
-    - 7.1
-    - nightly
+env:
+    global:
+        - PHPUNIT_FLAGS="-v"
```

Comment:
As we always run PHPUnit in verbose mode, we could move this one outside the `PHPUNIT_FLAGS` (as done for `COMPOSER_FLAGS` containing only the job-specific flags)